tests/install/worker_ay_validation.pm

```diff
@@ -0,0 +1,24 @@
+use Mojo::Base 'openQAcoretest';
+use File::Basename;
+use testapi;
+use utils;
+
+sub run {
+    switch_to_root_console;
+    my $profile = '/root/result.xml';
+    install_packages('autoyast2 libxml2-tools');
+    my $template_path = get_var('AUTOYAST');
+    my $profile_template = basename($template_path);
+    assert_script_run "wget $template_path";
+    # run-scripts=true is not used because the salt is not installed in the VM
+    enter_cmd "yast2 autoyast check-profile filename=$profile_template output=$profile run-erb=true";
+
+    assert_screen "check-autoyast-profile-ok", timeout => 300;
+    record_info "generated";
```

Comment:
?

Reply:
I want to make visible that the generation of the xml is done before the actual validation.

Reply:
Sorry, I still don't understand why. The test needs the profile to be generated anyway at this point. The test can just continue and if it can't continue then it will fail anyway. But maybe instead in https://github.com/os-autoinst/os-autoinst-distri-openQA/pull/198/files#diff-cf5d4733e495b63d23d5a7041a356f8b9e525d24fc65619db3548a02b425c7ebR18 you want to add a `fail_message`?

Reply:
I would expect the command to fail or the needle to mismatch anyway?

Reply:
https://github.com/os-autoinst/os-autoinst-distri-openQA/commit/2fd5ac2a27c2624d36005903929563109fa961d9 adds a fail_message and removes the record_info